android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -125,6 +124,7 @@ public void getMacAddress(Promise p) {
       constants.put("firstInstallTime", info.firstInstallTime);
       constants.put("lastUpdateTime", info.lastUpdateTime);
     } catch (PackageManager.NameNotFoundException e) {
+      constants.put("appVersion", "error");
```

Comment:
This sadly causes a BC break for people relying on the string "not available" being returned upon error :/

I'll try to introduce a nicer debug mode for 1.0.0 (#307)